core/src/main/scala/com/emaginalabs/kuronometer/core/model/BuildExecutionReport.scala

```diff
@@ -0,0 +1,26 @@
+package com.emaginalabs.kuronometer.core.model
+
+case class ProjectInformation(name: Option[String], description: Option[String])
+
+case class ProjectVersion(group: Option[String], version: String)
+
+case class ProjectDefinition(info: ProjectInformation, version: ProjectVersion)
```

Comment:
The ``info`` and ``version`` params are optional, not the values inside the ``ProjectInformation`` and ``ProjectVersion`` case classes. [Here](https://github.com/pedrovgs/Kuronometer/blob/master/kuronometer-core/src/test/resources/reportAnonymousBuildExecutionRequest.json) you have an anonymous report and [this](https://github.com/pedrovgs/Kuronometer/blob/master/kuronometer-core/src/test/resources/reportBuildExecutionRequest.json) is a regular report with all the information we can collect from client side code.